apps/docs-snippets/src/guide/contracts/minted-token-asset-id.test.ts

```diff
@@ -30,4 +30,16 @@ describe(__filename, () => {
     expect(mintedAssetId).toBeDefined();
     expect(txResult.transactionResult.isStatusSuccess).toBeTruthy();
   });
+
+  it('should create valid asset ID', async () => {
+    // #region create-asset-id-1
+    const subID = '0xc7fd1d987ada439fc085cfa3c49416cf2b504ac50151e3c2335d60595cb90745';
+
+    const assetId = createAssetId(contract.id.toB256(), subID);
+    // #endregion create-asset-id-1
+    const { value } = await contract.functions.echo_asset_id_comparison(assetId).simulate();
```

Comment:
@bolajahmad The function `echo_asset_id_comparison` does not exist on the contract used within this code snippet. 

We need to use `mint_coins` instead

Reply:
Added new function `echo_asset_id_input` and adjusted accordingly :)